packages/iobio-react-components/src/utils/elasticMetadataClient/index.mts

```diff
@@ -39,7 +40,11 @@ import {
 
 const authKey = process.env.ES_AUTH_KEY;
 const esHost = process.env.ES_HOST_URL;
-if (!(authKey && esHost)) throw new Error('Required .env configuration values are missing');
+if (!(authKey && esHost)) throw new Error('Required ElasticSearch .env configuration values are missing');
+
+const scoreApiUrl = process.env.SCORE_API_URL;
+const scoreApiDownloadPath = process.env.SCORE_API_DOWNLOAD_PATH;
+if (!(scoreApiUrl && scoreApiDownloadPath)) throw new Error('Required Score .env configuration values are missing');
```

Comment:
This branch was originally based off of: https://github.com/overture-stack/iobio-components/pull/78/files

I've kept a few changes related to reorganizing the Score setup as part of this branch
Score utils have uses both in the UI context & Node context